devbuild.sh

```diff
@@ -68,9 +68,22 @@ fi
 # Source the README file for this platform/compiler combination, then build the code
 . $ENV_FILE
 
+cp src/CMakeLists.txt_model src/CMakeLists.txt
 mkdir -p ${BUILD_DIR}
 cd ${BUILD_DIR}
-cmake .. -DCMAKE_INSTALL_PREFIX=..
+cmake .. -DCMAKE_INSTALL_PREFIX=.
 make -j ${BUILD_JOBS:-4}
+mv bin ../.
+
+cd ..
+cp src/CMakeLists.txt_DA src/CMakeLists.txt
+. ${ENV_FILE}_DA
+rm -fr ${BUILD_DIR}
+mkdir -p ${BUILD_DIR}
+cd ${BUILD_DIR}
+cmake .. -DCMAKE_INSTALL_PREFIX=.
+make -j ${BUILD_JOBS:-4}
+cd bin
+cp * ../../bin/.
 
```

Comment:
Is all this necessary?  It's not clear to me why the /bin directory is being moved here.  The build directory is created, deleted. then recreated again.  Can the GSI utilities not be built within the main build and bin directory?

Reply:
The original "-DCMAKE_INSTALL_PREFIX=.."  install everything in main directory, which messed up with the code. One thing cmake should do is complete separate the build and code. So I think we should use "-DCMAKE_INSTALL_PREFIX=." for a clean build.

The workflow is looking for a bin directory under the main directory. That is why the bin has to be moved up to main. 

GSI needs difference modules and environmental setups. I cannot figure out a way to do it in one build and has to build twice, one for model and one for GSI.

Reply:
@hu5970, thanks for making these changes.  One last question.  The workflow should look for /bin in the ufs-srweather-app directory, which is where devbuild.sh sits.  Why is is looking for it in ufs-srweather-app/../?

Reply:
install to system root is not a good and clean way. I will return this back for this PR but will talk about this on our RRFS meeting.